web-console/knikubevirt/vm-list/vm-list.md

```diff
@@ -0,0 +1,85 @@
+# VM List
+
+![List of VMs](img/0-0-main.png)
+
+The VM list contains columns for the following:
+- Name
+- Namespace
+- Status
+  - Running, Off, Error (with custom messages), Cloning, Migrating, Importing, Warning, Pending (with changes that will apply after restart), Unknown
+- Created
```

Comment:
I may recommend dropping created unless there's a specific reason it is needed for VMs. We only show created for Secrets and Config Maps. If it needs to stay, can you update the format to match that of secrets and config maps?